{{cookiecutter.project_slug}}/compose/local/docs/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.8-slim-buster
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN apt-get update \
+    # dependencies for building Python packages
+    && apt-get install -y build-essential \
+    # psycopg2 dependencies
+    && apt-get install -y libpq-dev \
+    # Translations dependencies
+    && apt-get install -y gettext \
+    # Uncomment below to enable Sphinx output to latex and pdf
+    # && apt-get install -y texlive-latex-recommended \
+    && apt-get install -y texlive-fonts-recommended \
+    && apt-get install -y texlive-latex-extra \
+    && apt-get install -y latexmk \
```

Comment:
Do we need these if we don't use latex?

Reply:
I think it was decided [to drop it](https://github.com/pydanny/cookiecutter-django/pull/2555#discussion_r426654280).

Reply:
I commented line 14 -- I meant to comment out lines 14-17 with the note on line 13.
I've seen little notes like that in the project elsewhere.
Should I just delete instead?
I'll fix it in either direction.

Reply:
Don't worry about squashing, we can do it at merge time.

I don't mind either way, really. I don't use latex anymore, so I'd be in favour of removing them, but that's subjective.

Reply:
I think @hanhanhan question was whether to keep the lines but comment them out. Personally, I rather delete lines then comment them because, well - that's what git is for! 😄

Reply:
Git history won't be preserved into projects that will be generated, only in the template code.

Reply:
Good point - didn't think it from that perspective. This is super helpful PR and I tried out on a demo project recently and I can report that it worked fine with few custom modifications I need for my local docker setup (not relevant to the project).

Thanks a lot @hanhanhan !